packages/shared/src/node-apis/redis.ts

```diff
@@ -46,13 +43,15 @@ type PolkadotBroadcast = ChainBroadcast<'Polkadot'>;
 type BitcoinBroadcast = ChainBroadcast<'Bitcoin'>;
 type Broadcast = ChainBroadcast<Chain>;
 
-const mempoolTransaction = z.object({
-  confirmations: number,
-  value: number.transform((value) =>
-    new BigNumber(value).shiftedBy(assetDecimals.BTC).toString(),
-  ),
-  tx_hash: string.transform((value) => `0x${value}` as const),
-});
+const mempoolTransaction = jsonString.pipe(
+  z.object({
+    confirmations: number,
+    value: number.transform((value) =>
+      new BigNumber(value).shiftedBy(assetDecimals.BTC).toString(),
```

Comment:
i think you changed this, do we still need the shift?

Reply:
fixed